ros2_nats_bridge:
  ros__parameters:
    UNIT_ID: "vehicle_id"
    NATS_SERVER_IP_PORT: "nats://192.168.1.154:4222"
    UNIT_TYPE: "platform"
    UNIT_NAME: "Black_Pacifica"
    EVENT_NAME: "UC3"
    LOCATION: "tfhrc"
    TESTING_TYPE: "Integration"